pages/windows/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Displays the name of or changes the current working directory.
+> More information: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cd.
+
+- Go to a directory in the same drive:
+
+`cd {{path/to/directory}}`
+
+- Display the name of the current directory:
+
+`cd`
+
+- Go up to the parent of the current directory:
+
+`cd ..`
+
+- Go to a directory in a different drive:
+
+`cd {{path/to/directory}} /d`
```

Comment:
Is `/d` here not a mistake? Is it some sort of a command flag, or some path?

Reply:
Nope, not a mistake. 👍 Windows traditionally used `/{flag}` for flags, rather than `--{flag}`.

`/d` is correctly used to change to another drive.

Reply:
Alright, thanks for explaining. I approved this now.

Reply:
Personally I'd put the `/d` directly after the `cd`, but that's just personal preference.